relay/http.go

```diff
@@ -300,6 +324,10 @@ func (b *httpBackend) post(buf []byte, query string) (response *http.Response, e
 	req.Header.Set("Content-Type", "text/plain")
 	req.Header.Set("Content-Length", fmt.Sprint(len(buf)))
 
+	if b.authorization != "" {
+		req.Header.Set("Authorization", b.authorization)
```

Comment:
This should use [`req.SetBasicAuth`](https://golang.org/pkg/net/http/#Request.SetBasicAuth).